app/models/claim.rb

```diff
@@ -343,4 +357,8 @@ def determine_student_loan_plan
       Claim::NO_STUDENT_LOAN
     end
   end
+
+  def passed_scheduled_payment_date?
+    scheduled_payment_date < Time.zone.now if scheduled_payment_date
+  end
```

Comment:
I don't think this method is used any more

Reply:
Thanks!